JS-Algorithms/challenges.js

```diff
@@ -118,9 +168,19 @@ Because the 12th index in the Fibonacci sequence is 144, and 144 has three digit
 ***** */
 
 const fibIndex = (n) => {
-  // YOUR CODE HERE...
+  let a = 1, b = 1, index = 2; // here i initialized the first two Fibonacci numbers and the index
```

Comment:
Although declaring multiple variables with the comma operator is totally valid, I would prefer to declare them in separated lines for readability and maintainability purposes.

Totally fine in this code challenge!